Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -191,6 +194,38 @@ public sealed partial class HumanoidProfileEditor : Control
 
             #endregion Species
 
+            // Parkstation-HeightSlider Start
+            #region Height
+
+            var prototype = _speciesList.Find(x => x.ID == Profile?.Species) ?? _speciesList.First();
+
+            _heightSlider.MinValue = prototype.MinHeight;
+            _heightSlider.MaxValue = prototype.MaxHeight;
+            _heightSlider.Value = Profile?.Height ?? prototype.DefaultHeight;
+            CHeightLabel.Text = Loc.GetString("humanoid-profile-editor-height-label", ("height", _heightSlider.Value));
+
+            _heightSlider.OnValueChanged += args =>
+            {
+                if (Profile is null)
+                    return;
+
+                prototype = _speciesList.Find(x => x.ID == Profile.Species) ?? _speciesList.First(); // Just in case
```

Comment:
I don't love this being done every time the slider is  touched. Is it really necessary?

Reply:
It performs fine, but I'll see if it's needed or if there's an alternative